tests/perf/srv_perf.py

```diff
@@ -0,0 +1,120 @@
+# This script can be used for performance testing of the DeepRank2 package, using the SRV query classes.
+import glob
+import os
+import time
+from os import listdir
+from os.path import isfile, join
+
+import numpy
+import pandas as pd
+
+from deeprank2.domain.aminoacidlist import (alanine, arginine, asparagine,
+                                            aspartate, cysteine, glutamate,
+                                            glutamine, glycine, histidine,
+                                            isoleucine, leucine, lysine,
+                                            methionine, phenylalanine, proline,
+                                            serine, threonine, tryptophan,
+                                            tyrosine, valine)
```

Comment:
how about

```suggestion
from deeprank2.domain import aminoacidlist as aa
```

and then calling them as `aa.serine`, etc below

Reply:
don't spend time on this either...